spec/features/items/destroy_spec.rb

```diff
@@ -2,29 +2,18 @@
 
 RSpec.describe 'item delete', type: :feature do
   describe 'when I visit an item show page' do
-    it 'I can delete an item' do
-      bike_shop = Merchant.create(name: "Brian's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
-      chain = bike_shop.items.create(name: "Chain", description: "It'll never break!", price: 50, image: "https://www.rei.com/media/b61d1379-ec0e-4760-9247-57ef971af0ad?size=784x588", inventory: 5)
-      visit "/items/#{chain.id}"
-
-      expect(page).to have_link("Delete Item")
-
-      click_on "Delete Item"
-
-      expect(current_path).to eq("/items")
-      expect("item-#{chain.id}").to be_present
-    end
+    # it 'I can delete an item' do #FUNCTION MOVED-LIMITED TO MERCHANT_ITEMS CONTROLLER - KM
```

Comment:
As a final thing we should decide if we should keep comments in there or delete lines.